src/diagnostics/logPointsClient.ts

```diff
@@ -202,9 +223,16 @@ export class MockLogpointsDebuggerClient extends LogPointsDebuggerClientBase imp
     }
 
     public attachProcess(siteName: string, affinityValue: string, publishCredential: WebSiteModels.User, data: IAttachProcessRequest): Promise<CommandRunResult<IAttachProcessResponse>> {
+        const jsonPayload = JSON.stringify({
```

Comment:
Maybe better to do the JSON.stringify inside the base64encode method on a JS object? That way the clients can send you any data instead of just a string?

Reply:
Sounds like a good idea to remove some duplication. will do.

Reply:
Done.